### ISSUE TITLE
Panic with using pgx

### ISSUE THREAD

**Issue by vivekjoshi108** [memwey/casbin-sqlx-adapter#7]
The second line in code causes panic:

```
opts := &sqlxadapter.AdapterOptions{
		DriverName: "pgx",
		DataSourceName: v.GetString("dsn"),
		TableName: "casbin_rule",
	}
a := sqlxadapter.NewAdapterFromOptions(opts)
```
```
panic: ERROR: syntax error at or near "`" (SQLSTATE 42601)

goroutine 1 [running]:
github.com/memwey/casbin-sqlx-adapter.(*Adapter).ensureTable(0xc000372820)
        /Users/vivek/work/pkg/mod/github.com/memwey/casbin-sqlx-adapter@v0.2.0/adapter.go:99 +0x115
github.com/memwey/casbin-sqlx-adapter.NewAdapterFromOptions(0xc000217f18, 0x18a9083)
        /Users/vivek/work/pkg/mod/github.com/memwey/casbin-sqlx-adapter@v0.2.0/adapter.go:168 +0x93
main.main()
```

**Comment by hsluoyz** [comment]
@memwey

**Comment by kilosonc** [comment]
@vivekjoshi108 It was caused by no such table in database.You could just create the table manually.

**Comment by kilosonc** [comment]
@vivekjoshi108 plz refer to https://github.com/memwey/casbin-sqlx-adapter/blob/master/adapter_test.go#L138

**Comment by hsluoyz** [comment]
@closetool can we automatically create the table?

**Comment by memwey** [comment]
This is caused by
```sql
"SELECT 1 FROM `%s` LIMIT 1"
```
postgresql doesn't support backquote, I will have a look at it

**Comment by memwey** [comment]
Solved in v0.2.1

**Comment by hsluoyz** [comment]
@vivekjoshi108 fixed by commit: https://github.com/memwey/casbin-sqlx-adapter/commit/160976e9907db8e9dbc5e89048ab0272ae0f2cb8
